src/main/scala/uk/gov/hmrc/hmrcfrontend/views/html/components/package.scala

```diff
@@ -62,9 +62,17 @@ package object components extends Utils with Aliases {
   @deprecated(message = "Use DI", since = "Play 2.6")
   lazy val HmrcNotificationBadge = new hmrcNotificationBadge()
 
+  type HmrcPageHeadingSection = hmrcPageHeadingSection
+  @deprecated(message = "Use DI", since = "Play 2.6")
+  lazy val HmrcPageHeadingSection = new hmrcPageHeadingSection()
+
   type HmrcPageHeading = hmrcPageHeading
   @deprecated(message = "Use DI", since = "Play 2.6")
-  lazy val HmrcPageHeading = new hmrcPageHeading()
+  lazy val HmrcPageHeading = new hmrcPageHeading(HmrcPageHeadingSection)
+
+  type HmrcPageHeadingWithInlineSection = hmrcPageHeadingWithInlineSection
+  @deprecated(message = "Use DI", since = "Play 2.6")
+  lazy val HmrcPageHeadingWithInlineSection = new hmrcPageHeadingWithInlineSection(HmrcPageHeadingSection)
```

Comment:
- [ ] remove these so as not to conflict with 1.0, can still merge in ahead I think